<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="nk_label_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>   ΝΕΑ ΚΡΑΤΗΣΗ   </string>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΑΠΟ  :  </string>
     </property>
    </widget>
    <widget class="QDateEdit" name="nk_apo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2015</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>ΕΩΣ   :  </string>
     </property>
    </widget>
    <widget class="QDateEdit" name="nk_ews">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2015</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΑΡΙΘΜΟΣ ΑΤΟΜΩΝ  :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nk_atoma">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="nk_eisagwgi_kratisis">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ΕΙΣΑΓΩΓΗ ΚΡΑΤΗΣΗΣ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nk_onoma">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nk_epitheto">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΕΠΩΝΥΜΟ  :</string>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΟΝΟΜΑ  :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nk_tilefwno">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΤΗΛΕΦΩΝΟ  :</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="nk_textBrowser">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="nk_euresi_dwmatiou">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ΕΥΡΕΣΗ ΔΩΜΑΤΙΟΥ</string>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>ΣΤΟΙΧΕΙΑ ΚΡΑΤΗΣΗΣ</string>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΔΙΑΘΕΣΙΜΑ ΔΩΜΑΤΙΑ:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nk_arithmos">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nk_label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΑΡΙΘΜΟΣ ΔΩΜΑΤΙΟΥ  :</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>571</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="ed_label_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>   ΕΠΟΠΤΕΙΑ ΔΩΜΑΤΙΩΝ  </string>
     </property>
    </widget>
    <widget class="QLabel" name="ed_label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΗΜΕΡΟΜΗΝΙΑ   :</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="ed_imerominia">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2015</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="ed_label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ΚΑΤΕΙΛΗΜΜΕΝΑ</string>
     </property>
    </widget>
    <widget class="QLabel" name="ed_label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ΔΙΑΘΕΣΙΜΑ</string>
     </property>
    </widget>
    <widget class="QLabel" name="ed_label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ΚΡΑΤΗΜΕΝΑ</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="ed_textBrowser_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="ed_textBrowser_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="ed_textBrowser_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>171</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ed_anazitisi">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ΑΝΑΖΑΤΗΣΗ</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>441</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="ea_label_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>   ΕΠΙΒΕΒΑΙΩΣΗ-ΑΚΥΡΩΣΗ ΚΡΑΤΗΣΗΣ   </string>
     </property>
    </widget>
    <widget class="QDateEdit" name="ea_imeromina">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2015</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="ea_label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΗΜΕΡΟΜΗΝΙΑ   :</string>
     </property>
    </widget>
    <widget class="QLabel" name="ea_label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΚΡΑΤΗΣΕΙΣ:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ea_akurwsi_kratisis">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>190</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ΑΚΥΡΩΣΗ ΚΡΑΤΗΣΗΣ</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="ea_textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>411</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ea_anazitisi_kratisis">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ΑΝΑΖΑΤΗΣΗ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ea_arithmos_1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ea_label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΑΡΙΘΜΟΣ ΔΩΜΑΤΙΟΥ  :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ea_epivevaiwsi_kratisis_1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CHECK-IN</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>441</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="ea_label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>   CHECK-IN ΧΩΡΙΣ ΚΡΑΤΗΣΗ   </string>
      </property>
     </widget>
     <widget class="QDateEdit" name="ea_ews">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2015</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QLineEdit" name="ea_onoma">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ea_label_11">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ΑΠΟ  :  </string>
      </property>
     </widget>
     <widget class="QLabel" name="ea_label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Α.Δ.Τ.  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="ea_label_12">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>ΕΩΣ   :  </string>
      </property>
     </widget>
     <widget class="QLabel" name="ea_label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ΕΠΩΝΥΜΟ  :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="ea_apo">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2015</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QLineEdit" name="ea_tautotita_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ea_epitheto">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ea_label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ΟΝΟΜΑ  :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ea_epivevaiwsi_kratisis_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CHECK-IN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ea_arithmos_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ea_label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ΑΡΙΘΜΟΣ ΔΩΜΑΤΙΟΥ  :</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="ea_tautotita_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ea_label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Α.Δ.Τ.  :</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="dx_label_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>   ΔΙΑΧΕΙΡΙΣΗ ΔΩΜΑΤΙΩΝ  </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dx_arithmos">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dx_atoma">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dx_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΑΡΙΘΜΟΣ ΔΩΜΑΤΙΟΥ  :</string>
     </property>
    </widget>
    <widget class="QLabel" name="dx_label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ΑΡΙΘΜΟΣ ΑΤΟΜΩΝ  :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dx_diagrafi_dwmatiou">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ΔΙΑΓΡΑΦΗ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dx_anazitisi">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ΑΝΑΖΑΤΗΣΗ</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="dx_textBrowser">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="dx_eisagwgi_dwmatiou">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ΕΙΣΑΓΩΓΗ ΝΕΟΥ</string>
     </property>
    </widget>
    <widget class="QLabel" name="dx_label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ή</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
